### DISCUSSION TITLE
Split issue in drum dryer (Drying)

### DISCUSSION THREAD

**Discussion by Uzair1shah** [BioSTEAMDevelopmentGroup/biosteam#164] [Q&A] [answered]
Hello,

I am working on modeling a biorefinery using BioSTEAM, and I used Drum Dryer from the available units repository ('drying'). When I simulate drum dyer, it splits the whole feedstock instead of just splitting the moisture/water from the feedstock, which consists of 9 different components. I have tried using component-wise split, but gives me error. The part of the code for drum dryer is as follows:

```
# Rotary Dryer
S107 = Stream('S107', phase='g') #Air
S108 = Stream('S108', phase='g') #Natural gas
S109 = Stream('S109', phase='s') #dry feedstock/solids
S110 = Stream('S110', phase='g') #hot air
S111 = Stream('S111', phase='g') #emissions
UO105 = units.DrumDryer('UO105', ins=(S105,S107,S108), outs=(S109,S110,S111), split = 0.5 )

# Simulation of UO105 (Rotary dryer)
UO105.simulate()
UO105.show(flow='kg/hr')
r5 = UO105.results()
UO105.diagram()
print(r5)
```
The input and output streams are as follows:
ins:
![image](https://github.com/BioSTEAMDevelopmentGroup/biosteam/assets/128337604/1c63ed04-543d-45c4-b6f1-036e98e73fd1)
outs:
![image](https://github.com/BioSTEAMDevelopmentGroup/biosteam/assets/128337604/cf3766b0-4986-4457-8dc3-760dc99a3086)

So, instead of adding moisture, in a separate stream, S110, it adds up moisture in the feedstock while splitting all the components of feedstock. I am not sure if it is a bug or I am doing something wrong. I would really appreciate if you can help me in this regard. Thanks a lot.

**Comment by yoelcortes** [comment, marked as answer]
@Uzair1shah,

Components defined in split go to the gas. Use the `moisture_content` parameter to adjust the moisture content of your dried solids. I just added a full example in the documentation:

https://biosteam.readthedocs.io/en/latest/API/units/drying.html

Quick tip: You can use the python help function to view documentation in your console (e.g. `help(biosteam.DrumDryer)`). If you are using Spyder, you can also highlight the object and `ctrl+I` will show you the documentation in a pane.

I hope this helps!
Thanks.
